### PR TITLE
 Avoid 2 unnecessary dependecies.

### PR DESCRIPTION
**Pull request by brussee**
Triblerd running as a service does not need to know the api version.

### CODE REVIEW

**Comment by devos50** [comment]
Add to whitelist

**Comment by brussee** [comment]
@whirm ready

File: Tribler/Test/Core/test_osutils.py [hunk 1]
```diff
@@ -18,38 +18,38 @@ def test_fix_filebasename(self):
         default_name = '_'
         win_name_table = {
             'abcdef': 'abcdef',
-          '.': default_name,
```
Comment (whirm) [inline Tribler/Test/Core/test_osutils.py]: please, remove unrelated code reformatting from the PR. A mass reformatting will happen soon.

**Comment by brussee** [comment]
@whirm: Ready
